tests/parser/functions/test_convert_to_uint256.py

```diff
@@ -221,6 +221,17 @@ def foo() -> uint256:
         InvalidLiteralException
     )
 
+    code = """
+@public
+def foo() -> uint256:
+    return convert(-(-(-27.2319)), uint256)
```

Comment:
Please add tests that check the ast output of the annotate_and_optimize_ast function.